package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "powerbi-visuals-globemap",
   "description": "GlobeMap",
-  "version": "2.2.2",
+  "version": "2.3.0",
```

Comment:
Do we still need `"@types/d3": "3.5.36",`?

Reply:
Pbi utils use @types/d3